mlflow/projects/submitted_run.py

```diff
@@ -0,0 +1,77 @@
+import multiprocessing
+import os
+import signal
+
+from mlflow.entities.run_status import RunStatus
+from mlflow.projects.pollable_run import monitor_run, maybe_set_run_terminated
+from mlflow.utils.logging_utils import eprint
+
+
+def _run_in_subprocess(target, args, **kwargs):
+    """
+    Runs a Python function as a child process. The function's output will be streamed
+    to the current process's stdout/stderr
+    :param target: Function to run
+    :param args: Iterable of arguments to pass to the function
+    :param kwargs: Additional arguments to pass to the `multiprocessing.Process` launched to run the
+                   function.
+    :return: The `multiprocessing.Process` used to run the function
+    """
+    def wrapper():
+        target(*args)
+    p = multiprocessing.Process(target=wrapper, args=[], **kwargs)
+    p.start()
+    return p
+
+
+class SubmittedRun(object):
+    """
+    Class exposing information about an MLflow project run submitted for execution. Note that the
+    run ID may be None if it is unknown, e.g. if we launched a run against a tracking server that
+    our local client cannot access - in this case it's also not possible to get the run's status.
+    """
+    # TODO: we handle the case where the local client can't access the tracking server to support
```

Comment:
Where do we handle this case? Maybe we can move the TODO there. Also in the doc above we refer to "run id" but that is not one of the parameters.

Reply:
We handle it in `run_id` and `get_status`, will move the comment there & clarify the docstring :)